src/opera/util/metadata_utils.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+#
+
+"""
+============
+metadata_utils.py
+============
+
+ISO metadata utilities for use with OPERA PGEs.
+
+"""
+
+from osgeo import gdal
+from osgeo import osr
+import mgrs
+
+def get_geographic_boundaries_from_mgrs_tile(mgrs_tile_name, verbose=False):
+    """
+    Returns the Lat/Lon min/max values that comprise the bounding box for a given mgrs tile region.
+
+    Parameters
+    ----------
+    mgrs_tile_name : str
+        MGRS tile name
+    Returns
+    -------
+    lat_min : float
+        minimum latitude of bounding box
+    lat_max : float
+        maximum latitude of bounding box
+    lon_min : float
+        minimum longitude of bounding box
+    lon_max : float
+        maximum longitude of bounding box
+    """
+ 
+    mgrs_obj = mgrs.MGRS()
+    lower_left_utm_coordinate = mgrs_obj.MGRSToUTM(mgrs_tile_name)
```

Comment:
This just occurred to me, but we should probably wrap this call in a try/except for when an unknown or otherwise invalid tile code is provided. This would make a good additional test case as well.